actions/install/README.md

```diff
@@ -0,0 +1,29 @@
+# slsa-verifier setup GitHub Action
+
+This action installs the SLSA verifier so you can verify SLSA provenance in your workflows.
+
+For more information about `slsa-verifier`, refer to [its documentation](https://github.com/slsa-framework/slsa-verifier#verification-of-provenance).
+
+For more information about SLSA in general, see [https://slsa.dev](https://slsa.dev).
+
+## Usage
+
+This action requires using GitHub-provided Linux runners.
+
+Add the following snippet to your Github workflow YAML file:
+
+```yaml
+uses: slsa-framework/slsa-verifier-installer@main
+with:
+    verifier-release: v1.3.0 # optional
```

Comment:
we always have to expect vX.Y.Z here right? no resolution of vX to latest or vX.Y to latest patch